wrapanapi/systems/rhevm.py

```diff
@@ -643,6 +643,14 @@ def deploy(self, vm_name, cluster, timeout=900, power_on=True, **kwargs):
         if 'ram' in kwargs:
             vm_kwargs['memory'] = int(kwargs['ram'])  # in Bytes
         vms_service = self.system.api.system_service().vms_service()
+
+        root_password = kwargs.pop('root_password', None)
+        authorized_ssh_keys = kwargs.pop('authorized_ssh_keys', None)
+        if root_password or authorized_ssh_keys:
```

Comment:
Since we're not really doing any processing here, how about just passing `kwargs.initialization` through to `types.Initialization` instantiation, so _any_ of the Instantiation instance variables can be used, instead of just these three.

Callers would build their own dictionary of instantiation kwargs.

Reply:
done